client/state/login/actions.js

```diff
@@ -121,14 +89,8 @@ function getErrorFromHTTPError( httpError ) {
 		};
 	}
 
-	code = get( httpError, 'response.body.data.errors.code' );
-	if ( code ) {
-		message = get( httpError, 'response.body.data.errors.message' );
-	} else {
-		// TODO: Remove when we're done with the PR15485 fallback.
-		code = get( httpError, 'response.body.data.errors[0]', false );
-		message = getErrorMessageFromErrorCode( code );
-	}
+	const code = get( httpError, 'response.body.data.errors.code' );
+	let message = get( httpError, 'response.body.data.errors.message' );
```

Comment:
I would move this line inside the `if` clause below since `message` will be overridden anyway if there is no `code`.